internal/provider/incident_workflow_resource_test.go

```diff
@@ -51,47 +55,78 @@ func TestAccIncidentWorkflowResource(t *testing.T) {
 						"incident_workflow.example", "condition_groups.0.conditions.0.param_bindings.0.array_value.0.literal", "closed"),
 				),
 			},
+			// Update step and check new state
+			{
+				Config: testAccIncidentWorkflowResourceConfig(&workflowTemplateOverrides{
+					StepSlackMessage: "Don't forget to write a postmortem!",
+				}),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"incident_workflow.example", "steps.0.param_bindings.1.value.literal",
+						"{\"content\":[{\"content\":[{\"text\":\"Don't forget to write a postmortem!\",\"type\":\"text\"}],\"type\":\"paragraph\"}],\"type\":\"doc\"}"),
```

Comment:
I think we probably want our example to be something that doesn't include templated text? Just cause it's ugly and kinda distracting in our docs, I think